RepairsApi/V2/Boundary/Request/ListGasServiceDataRequest.cs

```
@@ -44,10 +53,14 @@ public class ListGasServiceDataRequest
         [FromQuery(Name = "reason")]
         public string Reason { get; set; }
 
+        [FromQuery(Name = "isactive")]
```

What about if they specify a report type, and another parameter? Do you want to throw an error?

---

That's a good point

---

Addressed, PR description will be amended